war/src/main/webapp/scripts/prototype.js

```diff
@@ -1986,12 +1986,6 @@ Element.Methods = {
     return element;
   },
 
-  remove: function(element) {
-    element = $(element);
-    element.parentNode.removeChild(element);
-    return element;
-  },
```

Comment:
why was this removed? it's what is causing the ATH failures.

Reply:
It was interfering with the removal of the modal (I'm assuming because the modal doesn't have a parent?

Reply:
do we still need to remove this function now that you've changed the core components to not use it?
https://github.com/search?q=org%3Ajenkinsci+Element.remove%28+language%3AJavaScript&type=code&l=JavaScript&p=1

It looks like it _should_ be okay to remove.

Reply:
(Accidentally resolved this in case you missed it)

Reply:
If I'm understanding correctly, we still need to remove the function as it interferes with removing the modal.

Reply:
I tested locally and it seems to be working fine? not sure how it could be causing an issue if nothing is calling it.

Reply:
<img width="584" alt="image" src="https://user-images.githubusercontent.com/43062514/225252729-b56609e9-c57e-4c1f-a531-393fed767bd5.png">

With the remove() function restored I get the above error when closing the modal.

Reply:
weird I get that error in Safari but not chrome...

Maybe it has a parent in chrome, I guess we already hit a browser difference with this functionality.